tools/labs/templates/hello_assignment/sol/hello-world.c

```diff
@@ -0,0 +1,21 @@
+#include <linux/kernel.h>
+#include <linux/init.h>
+#include <linux/module.h>
+
+MODULE_DESCRIPTION("Hello World");
+MODULE_AUTHOR("Psoru Lesfo Rever");
+MODULE_LICENSE("GPL");
+
+
+static int hello_init(void)
+{
+	printk(KERN_DEBUG "Hello, World!\n");
```

Comment:
I want to avoid using printk where possible, it is now recommended to use pr_debug, pr_info, etc. I think we can replace this with pr_info.

Also, I think we should add a TODO comment here so that this line is removed when the skel is generated.